src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetHash.cs

```diff
@@ -133,6 +124,29 @@ protected override void ProcessRecord()
             }
         }
 
+        /// <summary>
```

Comment:
The proposed Xml documentation header contains errors. Since this method is not externally visible (private), the header is not required. Please reword or remove.

Reply:
Done